policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit_base_cap.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class me_property_tax_fairness_credit_base_cap(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Maine property tax fairness credit base cap"
+    definition_period = YEAR
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.me.tax.income.credits.fairness.property_tax
+        filing_status = tax_unit("filing_status", period)
+        dependents = tax_unit("ctc_qualifying_children", period)
+        capped_dependents = min_(dependents, 2)
```

Comment:
2 should be a parameter

Reply:
the value "2" did not show in the legal code or tax forms, the description they use is "more than 1"